.github/workflows/kentik-api.yml

```diff
@@ -83,6 +83,7 @@ jobs:
           pip3 install -r requirements.txt
           pip3 install -r requirements-dev.txt
           python3 setup.py grpc_stubs # building grpc stubs must go before building
+          python3 setup.py setup_client_version # set sdk api client version to the latest git tag
```

Comment:
`setuptools_scm` should do it automagically.

Reply:
This call updates the content of "version.py" file